### PR TITLE
Add support for PDF/X-6.

### PR DESCRIPTION
**Pull request by datalogics-josepha**
Add support for PDF/X-6.

### CODE REVIEW

**Comment by datalogics-robb** [comment]
@datalogics-josepha is this feature in P1z or P2a?

**Comment by datalogics-josepha** [comment]
2a

**Comment by datalogics-robb** [comment]
> 2a

Ok, then this PR should have the label for "merge at release" applied to it, if it won't work with the P1z release.

File: DocumentConversion/ConvertToPDFX/ConvertToPDFX.cpp [hunk 1]
```diff
@@ -4,7 +4,7 @@
 // ConvertToPDFX converts the input PDF to a PDF/X compliant PDF.
 //
 // Command-line:  <input-pdf> <convert-option>       (all parameters are optional moving from left to right)
-//        where convert-option is 'PDFX4', 'PDFX1a2001', 'PDFX32003'
+//        where convert-option is 'PDFX4', 'PDFX6', 'PDFX1a2001',or  'PDFX32003'
```
Comment (datalogics-aarroyo) [inline DocumentConversion/ConvertToPDFX/ConvertToPDFX.cpp:7]: lets add a space after `or`  so its not right next to the comma.

**Review by datalogics-aarroyo — CHANGES_REQUESTED** [review]
everything looks good just a minor edit needed

File: DocumentConversion/ConvertToPDFX/ConvertToPDFX.cpp (hunk 1 above)
Comment (datalogics-josepha) [reply DocumentConversion/ConvertToPDFX/ConvertToPDFX.cpp]: fixed

**Review by datalogics-aarroyo — APPROVED** [review]

**Pull request merged**